grable_frontend/src/views/ARestaurant.vue

```diff
@@ -1,21 +1,38 @@
 <template>
-  <div class="A Post">
+  <div class="ARestaurant">
```

Comment:
No whitespace?

Reply:
> No whitespace?

I renamed the class of the div to a more logical name.